src/arviz_stats/psense.py

```diff
@@ -19,14 +19,14 @@
 
 def psense(
     dt,
+    var_names=None,
+    filter_vars=None,
     group="prior",
+    coords=None,
     sample_dims=None,
+    alphas=(0.99, 1.01),
```

Comment:
Isn't `alphas[0]` basically ignored now? Looks like I could do `(0.1, 1.01)` and `(0.99, 1.01)` and get the same result (maybe messing up the labels?)

Reply:
alpha[0] is not passed to `power_scale_sense` I think it should. I have reported this to the priorsense main dev. And he will check this ASAP.